src/proto/grpc/testing/control.proto

```diff
@@ -46,8 +46,15 @@ enum ServerType {
 }
 
 enum RpcType {
-  UNARY = 0;
-  STREAMING = 1;
+  // First there are ping-pong RPCs in which the client sends a request and the
+  // server sends a response
+  UNARY = 0; // unary ping-pong RPCs
+  STREAMING = 1; // streaming ping-pong RPCs
```

Comment:
I'm good with that; I'd actually prefer it. The hiccup is that it will require changes across several languages that already implement these tests, but it should be fine at this point.